packages/web-twig/tests/components-fixtures/tabsDefault.twig

```diff
@@ -1,23 +1,23 @@
 <TabList>
-  <TabItem>
-    <TabLink isSelected id="pane1-tab" target="pane1">
-      Item selected
-    </TabLink>
-  </TabItem>
-  <TabItem>
-    <TabLink id="pane2-tab" target="pane2">
-      Item
-    </TabLink>
-  </TabItem>
-  <TabItem>
-    <TabLink href="https://www.example.com">
-      Item link
-    </TabLink>
-  </TabItem>
+    <TabItem>
+        <TabLink isSelected id="pane1-tab" data-toggle="tabs" target="pane1">
+            Item selected
+        </TabLink>
+    </TabItem>
+    <TabItem>
+        <TabLink id="pane2-tab" data-toggle="tabs" target="pane2">
+            Item
+        </TabLink>
+    </TabItem>
+    <TabItem>
+        <TabLink href="https://www.example.com">
+            Item link
+        </TabLink>
+    </TabItem>
 </TabList>
 <TabPane id="pane1" isSelected label="pane1-tab">
-  Pane 1
+    Pane 1
 </TabPane>
 <TabPane id="pane2" label="pane2-tab">
-  Pane 2
+    Pane 2
```

Comment:
is this right? Did we miss the indentations here?

Reply:
That is ok.